.travis.yml

```
@@ -9,6 +9,7 @@ php:
     - 7.1
     - 7.2
     - 7.3
+    - 7.4snapshot
```

PHP 7.4 has been released:

```suggestion
    - 7.4
```